# Define the goal
string space
string message_type
string content # if message_type is text, string, else json.
---
# Define the result
bool done
---
# Define a feedback message
string status
